### DISCUSSION TITLE
How to match foders instead of files?

### DISCUSSION THREAD

**Discussion by frankpatrick** [tfeldmann/organize#145] [Q&A] [answered]
I usually download zip files, unzip them right inside my downloads folder.  I don't see how to match folders olders than 1 day. I need a rule that matches folders older than 1 day and moves them.

**Comment by tfeldmann** [comment]
This is coming in the next version but not yet possible at the moment

**Comment by tfeldmann** [comment, marked as answer]
This is now built in 🎉
